datamodel/openapi/openapi-generator/src/test/java/com/sap/cloud/sdk/datamodel/openapi/generator/DataModelGeneratorIntegrationTest.java

```diff
@@ -84,6 +97,7 @@ private enum TestCase
         String methodBuilder = null;
         String methodBuild = null;
         String constructorVisibility = null;
+        String aiSdkConstructor = null;
```

Comment:
why is this String and used with `"true"` and `null`?

```suggestion
        boolean aiSdkConstructor = false;
```

Reply:
`String` and not `boolean` because it is a maven property.
`null` instead of `"false"` because the `true/false` is a lie, the real activation of this maven property is `absent/present` , no matter the value of the property.